Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -40,7 +40,7 @@ open class _ComposerVC<ExtraData: ExtraDataTypes>: _ViewController,
         /// The text of the input text view.
         public var text: String
         /// The state of the composer.
-        public var state: ComposerState
+        public let state: ComposerState
```

Comment:
Why do we need it as immutable?

Reply:
It is explained in the documentation, we want to only change the state by creating a new `mutating func` and create the `Content` again. This is to avoid like when editing a message, we change the state to `state = .quote` but forget to change the `editingMessage` and `quotingMessage` accordingly